services/github/github-repo-size.service.js

```diff
@@ -51,6 +43,8 @@ export default class GithubRepoSize extends GithubAuthV3Service {
 
   async handle({ user, repo }) {
     const { size } = await this.fetch({ user, repo })
-    return this.constructor.render({ size })
+    // note the GH API returns size in KiB
+    // so we multiply by 1024 to get a size in bytes and then format that in IEC bytes
+    return renderSizeBadge(size * 1024, 'iec', 'repo size')
```

Comment:
This one is a bit wierd. The GitHub API exposes a number of Kibibytes (although the API docs say "kilobytes") rather than a number of bytes.

There is one place in the GitHub UI where this number is exposed/formatted and that is on https://github.com/settings/repositories

To work an example, if I call https://api.github.com/repos/chris48s/geometry-to-spatialite I get size=7159. That's presented in the frontend as
![Screenshot at 2024-09-21 18-54-20](https://github.com/user-attachments/assets/d40211fa-0dca-4df7-bb0e-5096ad438b7d)
7159/1024 = 6.99